classes/transactions.php

```diff
@@ -53,7 +53,9 @@ class transactions {
      * @return int|string the id of transaction record or error string.
      * @deprecated
```

Comment:
This function is not used any more as it is deprecated
I only keep it because other plugins still used it and will be deleted soon

We could add this to the class balance_op